deepchem/feat/base_classes.py

```diff
@@ -93,27 +93,28 @@ def _featurize_callback(
   return featurizer._featurize(mol_pdb_file, protein_pdb_file)
 
 
-class ComplexFeaturizer(Featurizer):
+class ComplexFeaturizer(object):
```

Comment:
Stop inheritng the Featurizer class because of type consistency. But, I think ComplexFeaturizer also should take one arguments like Featurizer.

In my image 


```python


class ComplexFeaturizer(object):
  def featurize(self, molecule_complexes: Iterable[Tuple[str, str]]):
  # some implementation

  def _featurize(self, molecule_complex: Tuple[str, str]):
      mol_file, protein_pdbs = molecule_complex
      # some implementation
```

Reply:
> I think conceptually, ComplexFeaturizer should inherit from Featurizer

I agreed! 

> I'll make both of these changes in a subsequent PR.

Thanks!